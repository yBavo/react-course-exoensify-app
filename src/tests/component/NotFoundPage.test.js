import React from 'react'
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import NotFoundPage from '../../component/NotFoundPage'


configure({ adapter: new Adapter() });

test('Should render NotFoundPage', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
})