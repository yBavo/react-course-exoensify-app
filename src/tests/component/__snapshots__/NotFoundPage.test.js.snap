// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFoundPage 1`] = `
<div>
  <h1>
    404 - Page not Found!
  </h1>
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
